### PR TITLE
adding support for lambda ci

### PR DESCRIPTION
**Pull request by alexandra-strojny**

### CODE REVIEW

**Comment by alexandra-strojny** [comment]
checked with the gitlab ci lint and an example (email-worker in notifications hub)

File: lambda-ci-resource.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+include:
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/base-ci-resource.yaml
```
Comment (zmarois) [inline lambda-ci-resource.yaml:2]: I'm not sure you can do multi-level includes. I think the file which includes this file needs to just include both.

Should be able to test this by referring directly to the commit locked url for  this file

https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/b75241dfea814275e813a47b19715e6bb44d8e57/lambda-ci-resource.yaml

File: lambda-ci-resource.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+include:
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/base-ci-resource.yaml
+
+publish:
+  stage: publish
+  script:
+    - echo "No-op publish for lambda deploy"
+
+deploy-int:
+  extends: .deploy
```
Comment (zmarois) [inline lambda-ci-resource.yaml:10]: Not sure you can use yaml inheritance unless the thing you are inheriting from is in this file.

**Review by smagidson — APPROVED** [review]

**Review by zmarois — APPROVED** [review]

File: lambda-ci-resource.yaml (hunk 2 above)
Comment (smagidson) [reply lambda-ci-resource.yaml:10]: This isn't YAML inheritance, though... it's GitLab's special inheritance.

Comment (zmarois) [reply lambda-ci-resource.yaml:10]: Ah, I see

https://docs.gitlab.com/ee/ci/yaml/#using-extends-and-include-together

Thanks!

File: lambda-ci-resource.yaml (hunk 1 above)
Comment (alexandra-strojny) [reply lambda-ci-resource.yaml:2]: I tested it with email-worker's ci lint checker and I saw the no op for publish, but maybe that's not completely correct?

**Pull request merged**